### PR TITLE
Updates the overview section of the API documentation

### PR DESCRIPTION
**Pull request by hlmnrmr**
Take 1
Relates to SD-2226

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

File: apiary.apib [hunk 1]
```diff
@@ -3,34 +3,99 @@ HOST: http://api.superdesk.org/v1/
...
 
 # Current Version
 
-TODO
+We are currently developing the __Superdesk Content API v1__ version, the release
+date is yet to be announced.
 
-# Content Schema
+When working with the __Superdesk Content API__, you must explicitly request the
+version of the API you want to use via URL, as in the following request:
```
Comment (mugurrus) [inline apiary.apib:31]: Version is not in the URL yet

File: apiary.apib [hunk 2]
```diff
@@ -3,34 +3,99 @@ HOST: http://api.superdesk.org/v1/
...
-TODO
+```
+https://api.superdesk.org/v1/items
+```
+
+# Content Model
+
+Content in Superdesk is organized following the __News Architecture__ schema defined as
+part of the [NewsML-G2](https://iptc.org/standards/newsml-g2/) standard for exchanging
+news and news management.
```
Comment (mugurrus) [inline apiary.apib]: Here we should specify the format we use (ninjs) and the fact that it's a flattened NewsML2

**Comment by mugurrus** [comment]
:+1: (except my comments)

File: apiary.apib (hunk 1 above)
Comment (hlmnrmr) [inline apiary.apib:31]: Right. We'll keep it as is for now though. It will get updated if needed.

**Pull request merged**